docs/source/starthere/fundamentals.rst

```diff
@@ -0,0 +1,240 @@
+NeMo fundamentals
+=================
+
+On this page we will go a little bit deeper on how NeMo works, to give you a good foundation for using NeMo for your :ref:`desired usecase <where_next>`.
+
+NeMo Models
+-----------
+
+A NeMo "model" includes all of the below components wrapped into a singular, cohesive unit:
+
+* neural network architecture,
+
+* dataset & data loaders,
+
+* preprocessing of input data & postprocessing of model outputs,
```

Comment:
remove comma and ampersand.

* preprocessing of input data and postprocessing of model outputs